Marlin/Marlin_main.cpp

```diff
@@ -5084,91 +5079,96 @@ inline void gcode_G28() {
 
         setup_for_endstop_or_probe_move();
 
-        test_precision =
-          #if ENABLED(DELTA_CALIBRATE_EXPERT_MODE)
-            // Expert mode : forced end at std_dev < 0.1
-            (verbose_level == 3 && zero_std_dev < 0.1) ? 0.0 :
-          #endif
-          zero_std_dev
-        ;
-
+        test_precision = zero_std_dev;
         float z_at_pt[13] = { 0 };
-
         iterations++;
 
         // probe the points
 
         int16_t center_points = 0;
 
-        if (probe_points != 3) {
+        if (abs(probe_points) != 3 &&  abs(probe_points != 6)) {                                 // probe centre
```

Comment:
Since this uses `abs(probe_points)` continually, please create a `const uint8_t` for it and use that instead.